### PR TITLE
Ping_fdp, and CRC import scripts and hook for bbmri-testnn added to repo

### PR DESCRIPTION
**Pull request by dtroelofsprins**
#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [ ] Code unit/system tested
- [ ] User documentation updated
- [x] Clean commits
- [ ] Updated CHANGELOG.md

### CODE REVIEW

**Comment by dtroelofsprins** [comment]
[Include all BBMRI-scripts on Github (ping_fdp and get CRC data on BBMRI-NL)](https://trello.com/c/vPlSYqaW/1102-include-all-bbmri-scripts-on-github-pingfdp-and-get-crc-data-on-bbmri-nl)

File: scripts/import_CRC_data.py [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+
+    data = crc_session.get(entity_type_id, batch_size=10000, uploadable=True)
+    df_crc = pd.DataFrame(data)
+
+    if entity_type_id == "eu_bbmri_eric_collections":
+        with printer.indentation():
+            printer.print("Modify CRC collections")
+            with printer.indentation():
+                printer.print("Replace unknown data_categories by OTHER")
+                for unk_type in [
```
Comment (jelmerveen) [inline scripts/import_CRC_data.py:41]: what is 'unk', short for unknown?

File: scripts/import_CRC_data.py [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+                    "CMS_STATUS",
+                    "NGS_RNAbased",
+                    "ORGANOID_DRUG_RESPOSE_DATA",
+                    "PATIENT_REPORTED_OUTCOMES",
+                    "treatment_data",
+                    "diagnostic_data",
+                    "TUMOR_STROMA_RATIO",
+                ]:
+                    df_crc["data_categories"] = df_crc["data_categories"].apply(
+                        lambda x: [i.replace(unk_type, "OTHER") for i in x]
```
Comment (jelmerveen) [inline scripts/import_CRC_data.py:62]: readability is a bit poor. maybe give a better name to x and possibly i

File: scripts/import_CRC_data.py [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+                    "CTCs",
+                    "H&E_SLIDES",
+                    "SWAB",
+                    "TMA",
+                    "CIRCULATING_ENDOTHELIAL_CELL",
+                    "ORGANOIDS",
+                    "TISSUE FRESH",
+                ]:
+                    df_crc["materials"] = df_crc["materials"].apply(
+                        lambda x: [i.replace(unk_type, "OTHER") for i in x]
```
Comment (jelmerveen) [inline scripts/import_CRC_data.py:82]: readability

File: scripts/import_CRC_data.py [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+                    "CLINICAL_EFFECTIVENESS_TRIAL",
+                    "TWICS",
+                    "MULTICENTER",
+                    "OBSERVATIONAL_STUDY",
+                    "EXPLORATORY_STUDY",
+                    "CLINICAL_INTERVENTION",
+                    "DOSE_ESCALATION_STUDY",
+                ]:
+                    df_crc["type"] = df_crc["type"].apply(
+                        lambda x: [i.replace(unk_type, "OTHER") for i in x]
```
Comment (jelmerveen) [inline scripts/import_CRC_data.py:101]: readability

File: scripts/import_CRC_data.py [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+                del row[column]
+
+            if column in int_columns and column in row:
+                row[column] = int(row[column])
+
+    return uploadable_data
+
+
+# Initiate the two sessions
+crc_session = EricSession(url="https://crc.molgeniscloud.org")
```
Comment (jelmerveen) [inline scripts/import_CRC_data.py:129]: urls in config?

**Review by jelmerveen — CHANGES_REQUESTED** [review]
Looks pretty solid to me! Just a few readability points and an optional config addition

File: scripts/import_CRC_data.py (hunk 1 above)
Comment (dtroelofsprins) [reply scripts/import_CRC_data.py:41]: Yes, indeed, I've replaced it by unknown_type

File: scripts/import_CRC_data.py (hunk 3 above)
Comment (dtroelofsprins) [reply scripts/import_CRC_data.py:82]: Replaced x by materials and i by material. Makes more sense now I guess.

File: scripts/import_CRC_data.py (hunk 4 above)
Comment (dtroelofsprins) [reply scripts/import_CRC_data.py:101]: Replaces x by types and i by coll_type. Makes more sense now I think.

File: scripts/import_CRC_data.py (hunk 2 above)
Comment (dtroelofsprins) [reply scripts/import_CRC_data.py:62]: Replaced x by categories and i by category. Makes more sense now I think.

File: scripts/import_CRC_data.py (hunk 5 above)
Comment (dtroelofsprins) [reply scripts/import_CRC_data.py:129]: I was already struggling with this a bit, as config was not really an option. But coming to think about this again now, I got the idea of using an extra script parameter for this.

**Pull request merged**